requirements.txt

```diff
@@ -1,12 +1,11 @@
-boto3==1.20.28
+boto3~=1.20
 gretel-client>=0.15.9
 gretel-synthetics[utils]
-jinja2==3.1.1
-networkx==2.6.3
-pandas==1.3.5
-pydantic==1.9.0
-requests==2.25.0
-scikit-learn==1.0.2
-smart-open[s3]==5.2.1
-sqlalchemy==1.4.42
-typing-extensions==4.3.0
+jinja2~=3.1
+networkx~=3.0
+pandas~=1.3
+pydantic~=1.9
+requests~=2.25
+scikit-learn~=1.0
```

Comment:
Relaxing the scikit requirement fixed the colab but we've been hitting with relational notebooks 😁 💯 .